en/includes/guides/agentic-ai/ai-agents/agent-credentials.md

```diff
@@ -0,0 +1,53 @@
+{{ product_name }} supports multiple authentication methods for AI agents, providing flexibility based on security requirements and implementation preferences.
+
+## Agent ID and Secret (Current)
+
+The default authentication method uses an Agent ID and secret pair.
+
+### Obtaining Credentials
+
+You will obtain the agent credentials at the time of registering the agent. In case you lost it or need a new secret, you can regenerate it by following the steps.
+
+1. On the {{ product_name }} Console, go to **Agents**.
+2. Click on the agent you want to update.
+3. Go to the Credentials tab.
+4. Under **Agent Secret**, click **Regenerate**.
+
+## Private Key JWT (Coming Soon)
+
+Private Key JWT authentication enables AI agents to authenticate with {{ product_name }} using a JSON Web Token (JWT) signed with a private key. This method eliminates the need for shared secrets, enhancing security by relying on asymmetric cryptography.
+
+### Key Requirements:
```

Comment:
Please fix the linter errors. (Trailing punctuation in headings and consecutive titles.)